packages/@okta/vuepress-site/docs/guides/sign-into-spa/define-callback/blazorwebasm/define-route.md

```diff
@@ -0,0 +1 @@
+Default callback route for Blazor App is pre-configured to be  `/authentication/login-callback`. Take a look at the guide [Customizing app routes](https://docs.microsoft.com/en-us/aspnet/core/blazor/security/webassembly/additional-scenarios?#customize-app-routes) if you need to change it.
```

Comment:
Is there a piece of code/file in the sample we can point devs to change the callback route?

Reply:
No, we don't have this sample